airflow/jobs/local_task_job.py

```diff
@@ -241,7 +241,6 @@ def _run_mini_scheduler_on_child_tasks(self, session=None) -> None:
                 include_upstream=False,
                 include_direct_upstream=True,
             )
-
```

Comment:
Could you revert this please so that we don't touch this file

Reply:
Done. Not sure how that snuck in there.